### PR TITLE
Issue503 Adding Nextera option for Shotgun kits

### PR DESCRIPTION
**Pull request by gwarmstrong**
I have added front-end support (WIP, pending back-end from @charles-cowart ) for multiple types of shotgun library prep kits.

With this update, `Nextera` and `KAPA HyperPlus kit` will be included in the interface.

Currently, all changes are concentrated in to `labcontrol.db.process.LibraryPrepShotgunProcess`. These changes should be able to be integrated once we have `kit_type` factored into the `create` method and `kit_type` property getter for the class.

### CODE REVIEW

File: labcontrol/gui/templates/library_prep_shotgun.html [hunk 1]
```diff
@@ -171,10 +174,21 @@ <h3>Shotgun library prep</h3>
...
+<!-- Kit type select -->
 <div class='form-group'>
-  <label class='control-label'><h4>KAPA HyperPlus kit:</h4></label>
-  <input type='text' id='kapa_hyperplus_kit-input' class='form-control' />
+<label class='control-label'><h4>Shotgun library prep kit:</h4></label>
+<select id='kit-type-select' class='form-control'>
+  <option selected disabled>Choose shotgun library prep kit...</option>
+  <option value='KAPA HyperPlus kit'>KAPA HyperPlus kit</option>
+  <option value='Nextera kit'>Nextera kit</option>
+  <!-- add new kit type options here -->
```
Comment (gwarmstrong) [inline labcontrol/gui/templates/library_prep_shotgun.html]: mention here where to add values into the database

**Comment by gwarmstrong** [comment]
@AmandaBirmingham @fedarko This PR should be ready for a review now.

On the front end, this PR adds a selection for 'Shotgun library prep kit' and refactor 'Kapa HyperPlus kit' to 'Lot ID' in the GUI:
Screen Shot 2019-09-09 at 5 21 04 PM

On the back end, this PR adds a table to the `labcontrol` schema `library_prep_shotgun_kit_type` which keeps track of the kit types, and adds a foreign key to `library_prep_shotgun_process` to keep track of which type of kit was used for each shotgun process.
Screen Shot 2019-09-09 at 3 23 23 PM

@AmandaBirmingham after talking with @fedarko , we were not sure what the best way to update the `labcontrol.html` for `dbschema` was, so I have only edited the `labcontrol.dbs`, do you have any suggestions?

**Review by AmandaBirmingham — COMMENTED** [review]
The front-end part of this looks great!  I think we may want to go a different direction for the back-end,  adding one more record to the labcontrol.reagent_composition_type (for Nextera) rather than bifurcating off a new table for shotgun library prep kit type.  Let's discuss more when we meet in person :)

**Comment by gwarmstrong** [comment]
Closing due to status of LabControl project.

**Pull request closed without merging**